client/Components/Settings/SettingsScreen.js

```diff
@@ -28,6 +29,23 @@ const SettingsScreen = ({navigation, route}) => {
     }
   };
 
+  // const sound1 = new Sound(require('./assets/SolXRloop.wav'),
```

Comment:
is this old code or future code? If it's code that will be used later, label it with // TODO:

Reply:
future code. it's not fully implemented yet